dev/plugins/hu.elte.txtuml.api.model.execution/src/hu/elte/txtuml/api/model/execution/diagnostics/protocol/GlobalSettings.java

```diff
@@ -9,4 +9,11 @@
 	 * Sets the VM property key which determines the DiagnosticsPlugin port
 	 */
 	public static final String TXTUML_DIAGNOSTICS_PORT_KEY = "txtUMLDiagnosticsPort";
+	
+	/**
+	 * Parameters for service over HTTP
+	 * NOT KEYS, only predefined parameters.
+	 */
+	public static final Integer TXTUML_DIAGNOSTICS_HTTP_PORT = 52525;
```

Comment:
We should make the HTTP port configurable from the active txtUML run configuration.